index.js

```diff
@@ -174,6 +174,12 @@ Parser.prototype._flush = function(done) {
     // ==========================
     for (var index in self.translations) {
         var key = self.translations[index];
+
+        // simplify ${dot.separated.variables} into just their tails (${variables})
+        key = key.replace(/\$\{([^}]+)\}/g, function(m, expr) {
```

Comment:
The primary reason to remove any dotted paths is to avoid breaking them up when constructing the json object, for example:
```
t(`root.${dot.separated}`)
// JSON file after parsing:
{"root": {"${dot": {"separated}": ""}}
```

This could of course be solved by modifying how the object is constructed, but I found that most times dot-separated expressions were present inside lookups they looked something like this:
```
t(`root.${this.state.account.type}`)
```
With `type` (and arguably `account`) being the only parts relevant to the translation.